### PR TITLE
Engineering principles updated November 2019

### PR DESCRIPTION
**Pull request by soulnafein**
I've tried to align our engineering principles to the one that designers/researchers/product managers have identified at the offsite in October 2019.
I've also merged a few that I think are covered by some of those principles.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,28 +26,30 @@ This page is in constant WIP, please refer to it regularly.
 
 # The carwow Engineering Principles
 
-
-__Build what matters__
-
-  * Engineering effort is a scarce commodity. It should only be applied to problems that “move the needle” for the company
-  * The biggest achievement of an Engineer at carwow is to provide positive business impact with no line of code written (maybe even removing some)
```
Comment (redroot) [inline readme.md]: What's the thinking behind removing this block @soulnafein ?

Comment (redroot) [reply readme.md]: ah nevermind i see its moved

**Review by johnwake — APPROVED** [review]
Nice update and good to see it being updated 😄

**Review by ccmdo — APPROVED** [review]

**Comment by redroot** [comment]
@soulnafein are you happy to merge this?

**Pull request closed without merging**